PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/DynamicThumbstick.rbxmx

```diff
@@ -88,7 +88,7 @@ function Thumbstick:Enable()
 	local humanoid = MasterControl:GetHumanoid()
 	if humanoid then
 		local shouldRevert = humanoid.AutoJumpEnabled == false
-		shouldRevert = shouldRevert and humanoid.DevTouchMovementMode == Enum.DevTouchMovementMode.UserChoice
```

Comment:
This does not exit on humanoid... rushing to get this in before code cutoff, I got sloppy.